From 444ca5e657498ed4848be4f0efdf6af2a163452a Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Tue, 11 Dec 2018 20:07:21 +0800
Subject: [PATCH] arm64: dts: rockchip: fix vbus-5v-gpios flags for rk3399
 platforms

Fusb302 driver uses gpiod_set_value() to control VBUS, so vbus-5v-gpios
flags should be used correctly in device tree.

Change-Id: Ie7f0d1d290750bbf4207c1bc5665e4a0427a7a97
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-box-rev1-android-6.0.dts | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts       | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts             | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android-6.0.dts | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts       | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts             | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi            | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi            | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi            | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-rock960-ab.dts           | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts            | 2 +-
 11 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-android-6.0.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-android-6.0.dts
index 89f53d059eb1..69028c683cf3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-android-6.0.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-android-6.0.dts
@@ -114,7 +114,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
index 1ae09d1ad145..3ad84d0d3730 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
@@ -119,7 +119,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
index 03a864fa8e27..9cf5690487f8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
@@ -114,7 +114,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android-6.0.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android-6.0.dts
index d294058195ca..b89ca93f463e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android-6.0.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android-6.0.dts
@@ -124,7 +124,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
index 99275c8261ae..8ea2e614b7b7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
@@ -129,7 +129,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts
index 2fe8cefb6abd..726dac1f0275 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts
@@ -124,7 +124,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
index 9e9a95e1c41b..f35d6ee56072 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
@@ -121,7 +121,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
@@ -334,7 +334,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb1_int>;
-		vbus-5v-gpios = <&gpio1 4 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 24 GPIO_ACTIVE_HIGH>;
 	};
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
index b893db8493ff..4b23c1fcadd8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
@@ -118,7 +118,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb1_int>;
-		vbus-5v-gpios = <&gpio1 4 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 24 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
@@ -359,7 +359,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
index 7d172ef4bb64..4c5f8a68d033 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
@@ -99,7 +99,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb1_int>;
-		vbus-5v-gpios = <&gpio1 4 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 24 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
@@ -340,7 +340,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rock960-ab.dts b/arch/arm64/boot/dts/rockchip/rk3399-rock960-ab.dts
index 601cd0c2351e..f2ad4d08c9cd 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rock960-ab.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rock960-ab.dts
@@ -504,7 +504,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
index 02b8ba7dcc94..e49be4035da6 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
@@ -618,7 +618,7 @@
 		reg = <0x22>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
-		vbus-5v-gpios = <&gpio1 RK_PA3 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 RK_PA3 GPIO_ACTIVE_HIGH>;
 		int-n-gpios = <&gpio1 RK_PA2 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
-- 
2.35.3

